src/pkg/cli/compose.go

```diff
@@ -121,8 +121,8 @@ func loadDockerCompose(filePath string, tenantId pkg.TenantID) (*types.Project,
 		ConfigFiles: []types.ConfigFile{{Filename: filePath}},
 		Environment: map[string]string{}, // TODO: support environment variables?
 	}, loader.WithDiscardEnvFiles, func(o *loader.Options) {
-		o.SetProjectName(projectName, true) // TODO: don't overwrite the declared project name in the compose file
-		o.SkipConsistencyCheck = true       // TODO: check fails if secrets are used but top-level 'secrets:' is missing
+		o.SetProjectName(projectName, false) // TODO: don't overwrite the declared project name in the compose file
```

Comment:
remove `todo:` (can leave the rest of the comment)

Reply:
fixed